README.md

```
@@ -4,17 +4,219 @@ Decimal floating point math implemented in Solidity/Yul.
+  - e.g. 0e5 and 0e15 have the same numeric value and so do 10e1 and 1e2 but
+    different numeric value always means different onchain value
+- Two numerically different values always format canonically to two different
+  strings
+- Every valid string has an associated numerical onchain value
+- Every numerical onchain value has a unique canonical string to represent it
+
+## Rounding vs. erroring vs. approximating
+
+Simply having an exact representation for every number we can write does not mean
+we have exact representations for all the outputs. E.g. 1/3.
+### Rounding
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)